Wasm: [Legacy] Build Failure: Wasm::SectionInfo default constructor could not be generated (branches: release/1.4, master)

https://ci2.dot.net/job/Microsoft_ChakraCore/job/release_1.4/job/daily_dev12_x64_debug/46/consoleFull#444524246a82fefab-f698-416f-8fca-58544c94cd4e

```
d:\j\workspace\daily_dev12_x---863a89ef\lib\wasmreader\WasmSection.h(34): error C2220: warning treated as error - no 'object' file generated [D:\j\workspace\daily_dev12_x---863a89ef\lib\Runtime\ByteCode\Chakra.Runtime.ByteCode.vcxproj]
15:38:23 d:\j\workspace\daily_dev12_x---863a89ef\lib\wasmreader\WasmSection.h(34): warning C4510: 'Wasm::SectionInfo' : default constructor could not be generated [D:\j\workspace\daily_dev12_x---863a89ef\lib\Runtime\ByteCode\Chakra.Runtime.ByteCode.vcxproj]
15:38:23           d:\j\workspace\daily_dev12_x---863a89ef\lib\wasmreader\WasmSection.h(27) : see declaration of 'Wasm::SectionInfo'
```

--- a/lib/WasmReader/WasmSection.cpp
+++ b/lib/WasmReader/WasmSection.cpp
@@ -8,6 +8,14 @@
 #ifdef ENABLE_WASM
 namespace Wasm
 {
+    SectionInfo::SectionInfo(
+        SectionFlag flag,
+        SectionCode precedent,
+        const char16* name,
+        const char* id,
+        const uint32 nameLength
+    ): flag(flag), precedent(precedent), name(name), id(id), nameLength(nameLength) {}
+
     SectionInfo SectionInfo::All[bSectLimit] = {
 #define WASM_SECTION(name, id, flag, precedent) {flag, bSect ## precedent, static_cast<const char16*>(_u(#name)), static_cast<const char*>(id), sizeof(#name)},
 #include "WasmSections.h"

--- a/lib/WasmReader/WasmSection.h
+++ b/lib/WasmReader/WasmSection.h
@@ -25,6 +25,7 @@ namespace Wasm
 
     struct SectionInfo
     {
+        SectionInfo(SectionFlag, SectionCode, const char16*, const char*, const uint32);
         SectionFlag flag;
         SectionCode precedent;
         const char16* name;

